File: Desafios em C#/1564 - Vai Ter Copa.cs
using System; 

//Developed by: @LucasMarcuzo

class URI {

    static void Main(string[] args) { 

        string str;
        
        while((str=Console.ReadLine())!= null)
        {
            if (str == null)
            {
                break;   
            }
            else          
            { 
                int x = int.Parse(str);
                if (x <= 0)
                {
                Console.WriteLine("vai ter copa!"); 
                }
                else
                {
                    Console.WriteLine("vai ter duas!");
                }
            }
        }
    }

}

File: Desafios em JavaScript/1564 - Vai Ter Copa.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

while(lines.length > 0)
{
    var x = parseInt(lines.shift());

    if (x.toString() === `NaN`)
    {
        break;   
    }
    else{
        if (x <= 0)
        {
            console.log("vai ter copa!"); 
        }
        else
        {
            console.log("vai ter duas!");
        }
    }
}